src/inverted-index.js

```diff
@@ -112,13 +118,35 @@ class InvertedIndex {
   search(index, term) {
```

Comment:
### Critical

This method is a oneliner that is very specific to `searchIndex` i.e won't be needed by any other method in this class, there's no need to have it as a separate method, it should remain in the `searchIndex` method.